### DISCUSSION TITLE
Is the firebase configuration safe?

### DISCUSSION THREAD

**Discussion by juandspy** [nunocoracao/blowfish#1068] [General]
Hi,
Looking at https://blowfish.page/docs/firebase-views/, it seems like in order to configure views and likes you have to expose the Firebase configuration in the `params.toml` and probably this will be embedded in the HTLM code (?). Isn't this unsafe? What is preventing others from using this config?

**Comment by nunocoracao** [comment]
Hello @juandspy no it should be all good. 
Those configs are meant to live client-side if you check Firebase docs. Firebase provides an additional layer of security on top of that with security rules on top of DBs and host filtering.

**Reply by juandspy** [reply]
TY!
